src/ngraph/op/fused/group_conv.cpp

```diff
@@ -178,3 +217,10 @@ void op::GroupConvolution::generate_adjoints(autodiff::Adjoints& adjoints, const
 {
     throw ngraph_error("NYI");
 }
+
+bool ngraph::op::GroupConvolution::has_groups_in_filters_shape() const
+{
+    // If filters_rank is (data_rank + 1), then filters are divided by groups on first
+    // dim.
+    return ((get_input_shape(0).size() + 1) == get_input_shape(1).size());
```

Comment:
This function should always return true, so it is useless. Weights tensor always contains group dimension by the sepcification. It is the main (and only) difference from the main convolution. It was a question to specification: whether we need to introduce another operation for GroupConvolution in addition to regular Convolution? We decided to introduce another op to avoid sub-classing of original Convolution. I wanted to avoid depending on rank to understand whether we have grouped convolution or not. In the same time we have enough other operations where several `modes` exist. Now lets follow current specification and organize code somehow that Convolution and GroupConvolution can reuse the same base code.